src/app/core/services/local-storage.service.ts

```diff
@@ -6,6 +6,7 @@ export const STORAGE_KEYS = {
   SIGN_UP_TUTORIAL_COMPLETED: 'signUpTutorialCompleted',
   SHOW_LIFETIME_BANNER: 'showLifetimeOffer',
   SHOW_CROWDCUBE_BANNER: 'showCrowdcubeBanner',
```

Comment:
if this `SHOW_CROWDCUBE_BANNER` is not gonna be used anymore remove it

Reply:
I don't know if we will use the banner or not